docs/devcontainers/containers.md

```diff
@@ -238,11 +238,11 @@ This implies trusting [the machine the Docker daemon runs on](/remote/advancedco
 
 VS Code's container configuration is stored in a [devcontainer.json](https://containers.dev/implementors/json_reference) file. This file is similar to the `launch.json` file for debugging configurations, but is used for launching (or attaching to) your development container instead. You can also specify any extensions to install once the container is running or post-create commands to prepare the environment. The dev container configuration is either located under `.devcontainer/devcontainer.json` or stored as a `.devcontainer.json` file (note the dot-prefix) in the root of your project.
 
-You can use any image, Dockerfile, or set of Docker Compose files as a starting point. Here is a simple example that uses one of the pre-built [Development Container images](https://github.com/devcontainers/images):
+You can use any image, Dockerfile, or set of Docker Compose files as a starting point. Here is a simple example that uses one of the pre-built [Development Container images](https://github.com/devcontainers/images/tree/main/src/typescript-node):
```

Comment:
I wonder if we want to point this to an index of the images. We don't have an index in MCR yet, but I suppose we could point at the tracking issue for now: https://github.com/devcontainers/images/issues/90

Reply:
I'd consider the devcontainers/images repo a comprehensive index for users trying to explore what images we offer. Is there a limitation to the /images repo that maybe I'm not considering?

Reply:
Yeah it doesn't help with the actual image names - you end up having to look at each README to see it. That may be fine for now tho.

Reply:
I think it's fine for now, and avoids users needing to read through and understand an additional issue. I also think it's not the total worst to point them to where they can learn more about each image (by exploring each README), so I'll leave this as-is at least for now.